metpy/plots/_util.py

```diff
@@ -31,17 +32,31 @@ def add_timestamp(ax, time=None, x=0.99, y=-0.04, ha='right', **kwargs):
         Relative y position on the axes of the timestamp
     ha : str
         Horizontal alignment of the time stamp string
+    high_contrast : bool
+        Outline text for increased contrast
 
     Returns
     -------
     `matplotlib.text.Text`
         The `matplotlib.text.Text` instance created
 
     """
+    if high_contrast:
+        text_args = {'color': 'white'}
+    else:
+        text_args = {}
+    text_args.update(**kwargs)
     if not time:
         time = datetime.utcnow()
     timestr = datetime.strftime(time, 'Created: %Y-%m-%dT%H:%M:%SZ')
-    return ax.text(x, y, timestr, ha=ha, transform=ax.transAxes, **kwargs)
+    text = ax.text(x, y, timestr, ha=ha, transform=ax.transAxes, **text_args)
+
+    if high_contrast:
+        # Make the text stand out even better using matplotlib's path effects
+        outline_effect = [patheffects.withStroke(linewidth=2, foreground='black')]
```

Comment:
Might be able to just include in `text_args` as `path_effects=[patheffects.withStroke(linewidth=2, foreground='black')]`.

That might simplify things a little.

Reply:
I like it as it'll let users specify custom path effects if they so desire.